examples/src/main/scala/org/apache/spark/examples/DFSReadWriteTest.scala

```diff
@@ -107,6 +110,13 @@ object DFSReadWriteTest {
 
     println("Writing local file to DFS")
     val dfsFilename = s"$dfsDirPath/dfs_read_write_test"
+
+    // delete file if exists
+    val fs = FileSystem.get(spark.sessionState.newHadoopConf())
+    if (fs.exists(new Path(dfsFilename))) {
+        fs.delete(new Path(dfsFilename), true)
+    }
+
```

Comment:
I need to delete the file upfront, otherwise if it is already there due to a previous run, tests will fail. 
I cannot delete it from within the test suite because permissions are 185:root. 
So I have to delete it from Spark.